src/components/WebexDataProvider/README.md

```diff
@@ -1,3 +1,29 @@
-# Webex Activity Data Provider Component
+# Webex Data Provider
 
-LALLI HELP ME HERE PLEASE. MY BRAIN IS FRIED RIGHT NOW!
+Webex Data Provider creates a context that providers data adapters to its child components.
```

Comment:
**provides** data adapters to its child ....

Reply:
Not sure I understand your request.

Reply:
you misspelled. read the sentence again.